app/assets/stylesheets/layouts/_dark_selectors.scss

```diff
@@ -87,21 +125,26 @@ table { border-color: #555; } // TODO: what does this style
 .icon-credit { background-color: $bg_color_icon_credit; }
 
 /* - tag items */
-.tag-item, tr.gallery-tags .tag-item { background-color: $bg_color_tag_item; color: $font_color_tag_item; }
+.tag-item, .gallery-tags .tag-item {
```

Comment:
I _think_ the `tr` on this `tr.gallery-tags` might have been necessary for rule-strength reasons – can you check it hasn't caused a regression? (You want to look at a gallery#show for a gallery with tags, and compare the before-and-after versions, in dark theme.)

Reply:
I think I had to put that back on, yeah, but I can check

Reply:
(It doesn't look to be back on all of them, but it is still on some.)

Reply:
It's not just dark, it's also:

https://github.com/Marri/glowfic/pull/857/files#diff-13f7d48b42c9bdae358798a016a706c2R50
https://github.com/Marri/glowfic/pull/857/files#diff-7de63ca325ddfc95b76d1c9746f2eb54R66